lib/Mojo/Message/Request.pm

```diff
@@ -11,7 +11,9 @@ has env    => sub { {} };
 has method => 'GET';
 has [qw(proxy reverse_proxy)];
 has request_id => sub {
-  substr sha1_sum($SEED . ($COUNTER = ($COUNTER + 1) % 0xffffff)), 0, 8;
+  substr( Digest::SHA::sha1_base64(
+    $SEED . ($COUNTER = ($COUNTER + 1) % 0xffffff)
+  ), 0, 8) =~ tr{+/}{-_}r;
```

Comment:
We import functions like `sha1_base64`.

Reply:
When was the non-destructive option added to `tr`? Does that match our Perl 5.10.1 requirement?

Reply:
Ah, Travis shows it was Perl 5.14. So this PR is not acceptable i'm afraid.